packages/fxa-payments-server/src/components/PaymentFormV2/index.tsx

```diff
@@ -144,15 +158,23 @@ export const PaymentForm = ({
       navigatorLanguages
     ));
   }
-
-  return (
-    <Form
-      data-testid="paymentForm"
-      validator={validator}
-      onSubmit={onSubmit}
-      className="payment"
-      {...{ onChange }}
-    >
+  const paymentSource = hasExistingCard ? (
+    <div className="card-details" data-testid="card-details">
+      <Localized
+        id="sub-update-card-ending"
+        vars={{
+          last: (customer as Customer).last4!,
+        }}
+      >
+        <div
+          className={`new-sub c-card unbranded ${customer?.brand?.toLowerCase()}`}
```

Comment:
Should this be something more like this? `${customer?.brand?.toLowerCase() || 'unbranded'}` 

Probably answering my own question: I think the more specific / later-defined CSS selectors win here, so it doesn't matter too much.